lib/maxmind/geoip2/reader.rb

```diff
@@ -56,6 +61,49 @@ def initialize(database, locales = ['en'], options = {})
       @locales = locales
     end
 
+    # Look up the IP address in the database.
```

Comment:
Will it be clear enough from the generated documentation which database each of these is for?

Reply:
This shows up under the method name, so probably. We could say a bit more though!